packages/components/src/components/hds/code-block/index.ts

```diff
@@ -31,8 +37,40 @@ import 'prismjs/components/prism-yaml';
 import 'prismjs/components/prism-markup-templating';
 import 'prismjs/components/prism-handlebars';
 
-export default class HdsCodeBlockIndexComponent extends Component {
-  @tracked prismCode = '';
+type Language =
+  | 'bash'
+  | 'go'
+  | 'hcl'
+  | 'json'
+  | 'log'
+  | 'ruby'
+  | 'shell-session'
+  | 'yaml';
```

Comment:
suggestions: I know it may look excessive, but for consistency on how we define argument values in components, could this be an enum in `types.ts` with an exported `HdsCodeBlockLanguages` type?

Reply:
Oh, totally! Updated.